### PR TITLE
Accessibility issues

### LINKED ISSUE #34: Want to improve the accessibility of the website
**Is your feature request related to a problem? Please describe.**
The website is not fully accessible which may create problems for some of the users on the internet. Like some images don't have alt text, skip heading level, etc.

**Describe the solution you'd like**
I want to improve the accessibility of the website. I will fix the heading levels, Contrast issues or add alt text wherever needed, etc.

**Describe alternatives you've considered**
Don't have an alternative solution for this issues

**Additional context**
image

### PR DESCRIPTION
**Pull request by muditchoudhary**
## What is the change?
Fixed the accessibility of the website

## Related issue?
closes: #34

## How was it tested?
Tested with wave accessibility extension tool.

## Checklist:
Before you create this PR, confirm all the requirements listed below by checking the checkboxes `[x]`:

-   [x] I have followed the [Contribution Guidelines]() while contributing.
-   [x] I have checked there aren't other open [Pull Requests](https://github.com/BugBustersCommunity/website/pulls) for the same update/change.
-   [ ] I have made corresponding changes to the documentation.
-   [x] I have tested the code before submission.
-   [ ] I have formatted the code . (You can use any html,css beautifier)
-   [x] My changes generates no new warnings.
-   [x] I'm HSOC22 contributor.
-   [ ] I have commented my code, particularly in hard-to-understand areas.

## GIF/video of the working of component you created:
image

### CODE REVIEW

**Review by siddhi-244 — APPROVED** [review]
Great work🥳🥳. You have found a unique issue and this shows you pay attention to small details too. Well done!

File: index.html [hunk 1]
```diff
@@ -172,24 +172,24 @@ <h3 style="margin: 0px 40%">But Why Us?</h3>
...
-                <button class="neo-button">
+                <button class="neo-button" aria-label="sidhe's linkedin handle">
                   <i class="fa-brands fa-linkedin-in"></i>
                 </button>
-                <button class="neo-button">
+                <button class="neo-button" aria-label="sidhe's instagram handle">
                   <i class="fa-brands fa-instagram"></i>
                 </button>
-                <button class="neo-button">
+                <button class="neo-button" aria-label="sidhe's github handle">
```
Comment (Aditi-Singh16) [inline index.html]: @muditchoudhary , its not correctly spelled

**Review by Aditi-Singh16 — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (muditchoudhary) [reply index.html]: ohh...I misspelled it, let me fix it

**Comment by muditchoudhary** [comment]
> Great workpartying_facepartying_face. You have found a unique issue and this shows you pay attention to small details too. Well done!

Thank you!! :)

File: index.html (hunk 1 above)
Comment (muditchoudhary) [reply index.html]: > @muditchoudhary , its not correctly spelled

Updated, see if everything is correct

**Review by Aditi-Singh16 — APPROVED** [review]

**Pull request merged**